tests/checkings.test.js

```diff
@@ -0,0 +1,96 @@
+    await CheckingPage.checkingReset()
+    await expect(CheckingPage.checkingNameInput).toHaveText('')
+})
+it('Should create a Standard Individual Checking', async ()=> {
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessNewCheckingForm()
+    await CheckingPage.chooseStandardChecking()
+    await CheckingPage.chooseIndividualChecking()
+    await CheckingPage.sendText(CheckingPage.checkingNameInput, 'Standard Individual Checking')
+    await CheckingPage.sendText(CheckingPage.checkingDepositInput, '25')
+    await CheckingPage.checkingSubmit()
+    await expect(CheckingPage.newCheckingConfirmation).toBeDisplayed()
+})
+it('Should switch checking', async ()=> {
+    await CheckingPage.activateSwitchChecking()
+    await expect(CheckingPage.checkingBody).toBeDisplayed()
+})
+it('Should create a Standard Joint Checking', async ()=> {
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessNewCheckingForm()
+    await CheckingPage.chooseStandardChecking()
+    await CheckingPage.chooseJointChecking()
+    await CheckingPage.sendText(CheckingPage.checkingNameInput, 'Standard Joint Checking')
+    await CheckingPage.sendText(CheckingPage.checkingDepositInput, '50.47')
+    await CheckingPage.checkingSubmit()
+    await expect(CheckingPage.newCheckingConfirmation).toBeDisplayed()
+})
+it('Should create a Interest Individual Checking', async ()=> {
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessNewCheckingForm()
+    await CheckingPage.chooseInterestChecking()
+    await CheckingPage.chooseIndividualChecking()
+    await CheckingPage.sendText(CheckingPage.checkingNameInput, 'Interest Individual Checking')
+    await CheckingPage.sendText(CheckingPage.checkingDepositInput, '1000000')
+    await CheckingPage.checkingSubmit()
+    await expect(CheckingPage.newCheckingConfirmation).toBeDisplayed()
+})
+it('Should create a Interest Joint Checking', async ()=> {
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessNewCheckingForm()
+    await CheckingPage.chooseInterestChecking()
+    await CheckingPage.chooseJointChecking()
+    await CheckingPage.sendText(CheckingPage.checkingNameInput, 'Interest Joint Checking')
+    await CheckingPage.sendText(CheckingPage.checkingDepositInput, '10000000')
+    await CheckingPage.checkingSubmit()
+    await expect(CheckingPage.newCheckingConfirmation).toBeDisplayed()
+})
+it('Should not create a checking because the deposit is less than 25', async ()=> {
+    await HomePage.displayCheckingMenu() 
+    await HomePage.accessNewCheckingForm()
+    await CheckingPage.chooseStandardChecking()
+    await CheckingPage.chooseIndividualChecking()
+    await CheckingPage.sendText(CheckingPage.checkingNameInput, 'Error')
+    await CheckingPage.sendText(CheckingPage.checkingDepositInput, '24.99')
+    await CheckingPage.checkingSubmit()
+    await expect(CheckingPage.newCheckingErrorText).toBeDisplayed()
```

Comment:
You can try more types of invalid input and make tests for it (maybe using a **data driven** approach as well), like negative numbers, characters, empty, etc

Reply:
I dont know how to assert it because a pop up appears.

Reply:
Assert that you stay in the same page instead of being redirected to the checkings-view, for example.